### PR TITLE
Add a schema for Fly.io's fly.toml

### PR DESCRIPTION
**Pull request by jsierles**
This PR aims to get VSCode schema validation for the [Fly.io](https://fly.io) deployment configuration file, [fly.toml](https://fly.io/docs/reference/configuration).

### CODE REVIEW

File: schemas/fly.toml.json [hunk 1]
```diff
@@ -0,0 +1,362 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "Repl.it config schema (.replit)",
+  "description": "https://fly.io/docs/reference/configuration",
+  "x-taplo-info": {
+    "authors": ["Joshua Sierles (https://github.com/jsierles)"],
+    "patterns": ["\\.*fly(.*)?\\.toml)?$"]
```
[the remark this answers is not in the document]
Comment (jsierles) [reply schemas/fly.toml.json]: Fixed

File: schemas/fly.toml.json [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+        "destination": {
+          "description": "The path at which the `source` volume should be mounted in the running app VM.",
+          "type": "string"
+        }
+      }
+    },
+    "experimental": {
+      "description": "Flags and features that are subject to change, deprecation or promotion to the main configuration.",
+      "type": "object",
+      "additionalProperties": false,
```
[the remark this answers is not in the document]
Comment (jsierles) [reply schemas/fly.toml.json]: Yeah, good idea. Fixed.

**Review by remorses — APPROVED** [review]

**Comment by tamasfe** [comment]
Hi, thanks for the PR.

> Thanks. Once this is merged, does anything else need to be done to get support in the VSCode plugin?

It should be available to the plugin once it is deployed to the website, so no action is required.

Unfortunately the CD for the website is currently broken and I haven't been able to take a proper look at it, I will merge this once it's fixed. I cannot say any ETA for it though, sorry.

**Comment by jsierles** [comment]
I tried fixing the website build issue. This seemed to work for me: https://github.com/tamasfe/taplo/pull/247

**Pull request merged**